model/Export/AbstractQTIItemExporter.php

```diff
@@ -208,6 +210,10 @@ public function export($options = [])
             $report->setType(\common_report_Report::TYPE_ERROR);
         }
 
+        // Set the right qti version
+        $content = preg_replace('/(http:\/\/www\.imsglobal\.org.+?)qtiv(\w\w\w)/', '$1qtiv' . $this->getQTIVersion(), $content);
```

Comment:
I'm afraid this might replace strings we don't want to replace. Can we be a bit more specific here?
Unit tests would be great.
`+?`: I don't understand the `?` after the `+` 
`(\w\w\w)`: the second `\w` can be replaced by a `p`

Reply:
Made regexp a bit more precise.
Add unit test.
+? - means lazy quantifier